thermal_processing.py

```diff
@@ -124,22 +126,39 @@ def one_candidate(candidates):
 def replace_ext(filename, ext):
     return filename.parent / (filename.stem + ext)
 
+def update_metadata(recording, api):
+    with open(recording["filename"], "rb") as f:
+        reader = CPTVReader(f)
+        metadata = {}
+        metadata["recordingDateTime"] = reader.timestamp.isoformat()
+        # Add device name when it can be processed on api server
+        # metadata["device_name"] = reader.device_name
+
+        count = 0.0
+        for frame in reader:
+            count += 1
+        metadata["duration"] = round(count / FRAME_RATE)
+    api.update_metadata(recording, metadata, not conf.do_classify)
+
 
 def main():
     api = processing.API(conf.api_url)
     s3 = processing.S3(conf)
 
     while True:
         try:
-            recording = api.next_job("thermalRaw", "toMp4")
+            recording = api.next_job("thermalRaw", "getMetadata")
```

Comment:
if `classify` is set to False then recordings will end up in `toMp4` and thermal_processing will never pick them up and take them further (even another thermal_processing instance with `classify` set to True). 

Why have the option to turn off classification anyway?

This needs some further thought.